### DISCUSSION TITLE
dokuwiki page isn't loading properly

### DISCUSSION THREAD

**Discussion by moontan** [combostrap/combo#16] [General]
Hi, I hope this is the right place to post this sort of issue.
I recently installed the Combo plugin for testing purposes in my dokuwiki instance. First I inserted some notes in one page using the `<inote>` tag which worked beautifully. Then I added some json data in order to create metatags for the page but after adding some info the page started behaving strangely, displaying a text field with scroll bars and all sorts of details on the wiki right where I had placed the json block. Then after a few attempts to alter the code and finally removing the block altogether, the page started to just display a blank white browser window when loaded while the Combo plugin was enabled. All other wiki pages seem fine.
I'll be happy to provide further debugging info.

**Comment by gerardnico** [comment]
The next release will come with a form to modify the metadata making the process easier.

We have seen blank pages due to a cache file that was created empty. We don't manage that but in the next release, we delete the cache file if it's empty.

If you have the stale plugin, you can delete easily the cache otherwise, you need to add '?purge=true' to the page url.

If you get a bad rendering, we just need the markup to test on our side.

**Comment by moontan** [comment]
Hi and thank you for your advice.
> If you have the stale plugin, you can delete easily the cache otherwise, you need to add '?purge=true' to the page url.

After trying `~~NOCACHE~~`, your `?purge=true` URL solution, and touching `conf/local.php` all without success, I got the Stale plugin and used it to no avail.
> If you get a bad rendering, we just need the markup to test on our side.

I'll attach several versions of the file from shortly before it stopped working to now. It could be that I violated some syntax standard as I've never worked with data structures in json before:
[rehearsal_studios.txt](https://github.com/ComboStrap/combo/files/7591329/rehearsal_studios.txt)
[rehearsal_studios.1637670299.txt.gz](https://github.com/ComboStrap/combo/files/7591341/rehearsal_studios.1637670299.txt.gz)
[rehearsal_studios.1637668086.txt.gz](https://github.com/ComboStrap/combo/files/7591342/rehearsal_studios.1637668086.txt.gz)
[rehearsal_studios.1637602700.txt.gz](https://github.com/ComboStrap/combo/files/7591343/rehearsal_studios.1637602700.txt.gz)
[rehearsal_studios.1637589426.txt.gz](https://github.com/ComboStrap/combo/files/7591344/rehearsal_studios.1637589426.txt.gz)
[rehearsal_studios.1637589308.txt.gz](https://github.com/ComboStrap/combo/files/7591345/rehearsal_studios.1637589308.txt.gz)
[rehearsal_studios.1637589192.txt.gz](https://github.com/ComboStrap/combo/files/7591346/rehearsal_studios.1637589192.txt.gz)
[rehearsal_studios.1637588911.txt.gz](https://github.com/ComboStrap/combo/files/7591347/rehearsal_studios.1637588911.txt.gz)
[rehearsal_studios.1637583457.txt.gz](https://github.com/ComboStrap/combo/files/7591348/rehearsal_studios.1637583457.txt.gz)
[rehearsal_studios.1637583333.txt.gz](https://github.com/ComboStrap/combo/files/7591349/rehearsal_studios.1637583333.txt.gz)
[rehearsal_studios.1637583142.txt.gz](https://github.com/ComboStrap/combo/files/7591350/rehearsal_studios.1637583142.txt.gz)
[rehearsal_studios.1637581751.txt.gz](https://github.com/ComboStrap/combo/files/7591351/rehearsal_studios.1637581751.txt.gz)
[rehearsal_studios.1637581685.txt.gz](https://github.com/ComboStrap/combo/files/7591352/rehearsal_studios.1637581685.txt.gz)
[rehearsal_studios.1637581538.txt.gz](https://github.com/ComboStrap/combo/files/7591353/rehearsal_studios.1637581538.txt.gz)
[rehearsal_studios.1637581408.txt.gz](https://github.com/ComboStrap/combo/files/7591354/rehearsal_studios.1637581408.txt.gz)
[rehearsal_studios.1637578316.txt.gz](https://github.com/ComboStrap/combo/files/7591355/rehearsal_studios.1637578316.txt.gz)
[rehearsal_studios.1637578052.txt.gz](https://github.com/ComboStrap/combo/files/7591356/rehearsal_studios.1637578052.txt.gz)
[rehearsal_studios.1636273939.txt.gz](https://github.com/ComboStrap/combo/files/7591357/rehearsal_studios.1636273939.txt.gz)
[rehearsal_studios.1636273836.txt.gz](https://github.com/ComboStrap/combo/files/7591358/rehearsal_studios.1636273836.txt.gz)
[rehearsal_studios.1636268711.txt.gz](https://github.com/ComboStrap/combo/files/7591359/rehearsal_studios.1636268711.txt.gz)
[rehearsal_studios.1636268634.txt.gz](https://github.com/ComboStrap/combo/files/7591360/rehearsal_studios.1636268634.txt.gz)
[rehearsal_studios.1636268194.txt.gz](https://github.com/ComboStrap/combo/files/7591361/rehearsal_studios.1636268194.txt.gz)
[rehearsal_studios.1636265139.txt.gz](https://github.com/ComboStrap/combo/files/7591362/rehearsal_studios.1636265139.txt.gz)
e.

**Comment by gerardnico** [comment]
Thanks for the markup.

The page is loading on our side:
https://drive.google.com/file/d/1-m0PpzCyPlL0rqwLWhIJhlmVVzU8TgxJ/view?usp=sharing

Comments:
  * most of the syntax is not a combo syntax (namely typography and openmaplayers, )
  * it takes a lot of time to render (but this is for now based on personal impression)
  * I didn't add the meta plugin (no found/search)

We can't show it to you on our demo website because the openmaplayers is adding a bunch of javascript for each page (ie it will increase our dokuwiki bundle on our platform and as we have a farm with several website, we can't add it)

I suggest that:
  * you take a copy of your page, 
  * clear the syntax
  * rebuild the syntax page by adding the syntax line by line or section by section

May be there is a conflict with the seo meta plugin, who knows ?

**Comment by gerardnico** [comment]
Also if you have access to the web log, may be there is more information in it.
If this is a non-compatibility issue, we can fix it.

**Comment by moontan** [comment]
> Also if you have access to the web log, may be there is more information in it. If this is a non-compatibility issue, we can fix it.

I'll try to talk to my webhosting provider and see what they can dig up.

> Thanks for the markup.
> 
> The page is loading on our side: https://drive.google.com/file/d/1-m0PpzCyPlL0rqwLWhIJhlmVVzU8TgxJ/view?usp=sharing
> 

I gave you a whole recent history of the page so you should be able to follow all changes I made playing around with the embedded Combo code. The screenshot looks like it's from the first, unzipped text file in the list. If so, then I don't think it contains any Combo plugin code as that's the last version I backupped before making the changes.

> Comments:
> 
> * most of the syntax is not a combo syntax (namely typography and openmaplayers, )
> * it takes a lot of time to render (but this is for now based on personal impression)
> * I didn't add the meta plugin (no found/search)
> 

I don't have the Meta plugin installed any more. I was actually pointed to Combo when I was looking for a more versatile replacement for this plugin and I uninstalled it pretty early.

> We can't show it to you on our demo website because the openmaplayers is adding a bunch of javascript for each page (ie it will increase our dokuwiki bundle on our platform and as we have a farm with several website, we can't add it)
> 
> I suggest that:
> 
> * you take a copy of your page,
> * clear the syntax
> * rebuild the syntax page by adding the syntax line by line or section by section
> 
> May be there is a conflict with the seo meta plugin, who knows ?

Okay. I'll fiddle around some more.
Thanks for looking into this!

**Reply by gerardnico** [reply]
A change of file does not have any impact on the output because the rendering takes only the last markup.
If you can get me the one that is causing the problem, I'll be happy to test it.

To not close the issue, I have migrated it as a discussion.
